patches/chromium/chore_expose_v8_initialization_isolate_callbacks.patch

```diff
@@ -25,17 +25,17 @@ index f417dca0103ddfd8ce6e3b3a4d033ea42133604e..d0ee37d147243858d997be230c12ea3e
      if (ContentSecurityPolicy* policy =
              execution_context->GetContentSecurityPolicy()) {
 diff --git a/third_party/blink/renderer/bindings/core/v8/v8_initializer.h b/third_party/blink/renderer/bindings/core/v8/v8_initializer.h
-index 932c6aad3df51dd6790e55bf708703767843dc5e..6865fd33e62f766c5a162ded8627c332bf2ed173 100644
+index 39d1351a4a8bbc95097640be54affec426abe73e..f1f124571db9f7c106c1ffe0bd11f39bf52bcd09 100644
 --- a/third_party/blink/renderer/bindings/core/v8/v8_initializer.h
 +++ b/third_party/blink/renderer/bindings/core/v8/v8_initializer.h
-@@ -69,8 +69,8 @@ class CORE_EXPORT V8Initializer {
+@@ -70,8 +70,8 @@ class CORE_EXPORT V8Initializer {
    static void MessageHandlerInWorker(v8::Local<v8::Message>,
                                       v8::Local<v8::Value>);
    static bool WasmCodeGenerationCheckCallbackInMainThread(
 -      v8::Local<v8::Context> context,
 -      v8::Local<v8::String> source);
 +    v8::Local<v8::Context> context,
 +    v8::Local<v8::String> source);
```

Comment:
This patch is no longer needed.  It was upstreamed via https://chromium-review.googlesource.com/c/chromium/src/+/2527943, but we missed it on a previous chromium roll.